share/examples/bitcoin.conf

```diff
@@ -46,7 +46,6 @@
 ##  all of them to open lots of connections.  Instead       ##
 ##  'connect' them all to one node that is port forwarded   ##
 ##  and has lots of connections.                            ##
-##       Thanks goes to [Noodle] on Freenode.               ##
```

Comment:
Not sure removing this is a good idea.

On one hand, [Noodle] is no longer a Freenode user (the NickServ database got reset)

On the other, it's not clear if this might be a copyright attribution?

Do we have a way to reach [Noodle] and ask?

Reply:
It was added in 68ba85fd750e94861440ef15047d571d6b518217 in 2014 with no other explanation.  Happy to drop the change if people think it's better to keep it.

Reply:
Done. It's not directly relevant.